src/Razor/Microsoft.NET.Sdk.Razor/integrationtests/Wasm/WasmPublishIntegrationTest.cs

```diff
@@ -61,31 +59,28 @@ public async Task Publish_WithDefaultSettings_Works()
                 serviceWorkerPath: Path.Combine("serviceworkers", "my-service-worker.js"),
                 serviceWorkerContent: "// This is the production service worker",
                 assetsManifestPath: "custom-service-worker-assets.js");
-
-            // When publishing without linker, we expect to have collation information present in mscorlib.dll
-            Assert.AssemblyContainsResource(result, Path.Combine(blazorPublishDirectory, "_framework", "_bin", "mscorlib.dll"), "collation.cjkCHS.bin");
```

Comment:
We have an issue - https://github.com/dotnet/aspnetcore/issues/22983 - to figure out if there's a replacement for this if any.